content/en/blog/2022/exponential-histograms/index.md

```diff
@@ -0,0 +1,297 @@
+it's likely you'll need to adjust the boundaries to better reflect your data.
+The view API helps, with mechanisms to select specific instruments and redefine
+the explicit bucket histogram aggregation bucket boundaries.
+
+In contrast, the only configurable parameter of exponential bucket histograms is
+the number of buckets, which defaults to 160 for positive values. The
+implementation automatically chooses the scale factor, based on the range of
+values recorded and the number of buckets available to maximize the bucket
+density around the recorded values. I can't overstate how useful this is.
+
+Exponential bucket histograms capture a high-density distribution of values **
+automatically adjusted for the scale and range of measurements**, with no
+configuration. The same histogram that captures nanosecond scale measurements is
+equally good at capturing second scale measurements. They retain fidelity
+regardless of scale.
+
+Consider the scenario of capturing HTTP request time milliseconds. With an
+explicit bucket histogram, you make guesses on bucket boundaries which you hope
+will accurately capture the distribution of values. But if conditions change and
+latency spikes, your assumptions might not hold and all values could be lumped
+together. Suddenly, you've lost visibility into the distribution of data. You
+know latency is high overall. But you can't know how many requests are high but
+tolerable versus terribly slow. In contrast, with an exponential bucket
+histogram, the scale automatically adjusts to the latency spikes to choose the
+optimal range of buckets. You retain insight into the distribution, even with a
+large range of measurement values.
+
+## Example scenario: explicit bucket histograms vs. exponential bucket histograms
+
+Let's bring everything together with a proper demonstration comparing explicit
+bucket histograms to exponential bucket histograms. I've put together
+some [example code](https://github.com/jack-berg/newrelic-opentelemetry-examples/commit/2681bf25518c02f4e5830f89254c736e0959d306)
+that simulates tracking response time to an HTTP server in milliseconds. It
+records one million samples to an explicit bucket histogram with the default
+buckets, and to an exponential bucket histogram with a number of buckets that
+produces roughly the same size
+of [OTLP](/docs/reference/specification/protocol/otlp/)
+-encoded, Gzip-compressed payload as the explicit bucket defaults. Through trial
+and error, I determined that ~40 exponential buckets produce an equivalent
+payload size to the default explicit bucket histogram with 11 buckets. (Your
+results may vary.)
+
+I wanted the distribution of samples to reflect what we might see in an actual
+HTTP server, with bands of response times corresponding to different operations.
+It will look something like this example:
+
+![target-probability-distribution-response-time](target-probability-distribution-response-time.webp)
```

Comment:
Ideally the image title gives a description on what you can see on that picture.

Reply:
I've tried to give all the images filenames that reflect what they represent. The image titles are therefore just the filenames. I can remove the dashes?